__tests__/fetchers.js

```diff
@@ -55,6 +55,22 @@ test('CopyFetcher.fetch', async () => {
   expect(contentFoo).toBe('bar');
 });
 
+test('GitFetcher.fetch fetchFromLocal not in network or cache', async () => {
+  const config = await createConfig();
+  const dir = await mkdir('git-fetcher');
+  const fetcher = new GitFetcher(dir, {
+    type: 'git',
+    reference: '/github.com/sindresorhus/beeper',
+    hash: '8beb0413a8028ca2d52dbb86c75f42069535591b',
+    registry: 'npm',
+  }, config);
+  try {
+    await fetcher.fetch();
+  } catch (err) {
+    expect(err.message).toBe(config.reporter.lang('tarballNotInNetworkOrCache', '/github.com/sindresorhus/beeper', '/github.com/sindresorhus/beeper'));
```

Comment:
I was wanting to do something like 
`config.getOfflineMirrorPath = jest.fn().mockReturnValueOnce('/some/dir/');`
To avoid the lookup of the dir  but if I do this flow complains
```
   60:   config.getOfflineMirrorPath = jest.fn().mockReturnValueOnce('/some/dir/');
         ^^^^^^^^^^^^^^^^^^^^ property `getOfflineMirrorPath`. Property not found in
   60:   config.getOfflineMirrorPath = jest.fn().mockReturnValueOnce('/some/dir/');
         ^^^^^^ Config
```
Someone know how I can avoid the error of flow?